tools/CoqMakefile.in

```diff
@@ -788,17 +788,23 @@ else
 TIMING_EXTRA =
 endif
 
-$(VOFILES): %.vo: %.v | $(VDFILE)
-	$(SHOW)COQC $<
-	$(HIDE)$(TIMER) $(COQC) $(COQDEBUG) $(TIMING_ARG) $(COQFLAGS) $(COQLIBS) $< $(TIMING_EXTRA)
+# can't make
+# https://www.gnu.org/software/make/manual/make.html#Static-Pattern
+# work with multiple target rules
+# so use eval in a loop instead
+define globvorule=
+
+  $(1).vo $(1).glob &: $(1).v | $(VDFILE)
```

Comment:
& is for https://www.gnu.org/software/make/manual/make.html#Multiple-Targets "Rules with Grouped Targets" btw

Reply:
This requires Gnu Make 4.3, which is the most recent minor release. Most users will not have it on their system. If it is now a requirement of `coq_makefile`, it should be advertised prominently.

Reply:
changed to do a feature check